### ISSUE TITLE
Low volume on Vita

### ISSUE THREAD

**Issue by JeffRuLz** [KuromeSan/GayMaker#3]
Every time I export a game with this program, the game's volume is much lower than other apps on my Vita. About half as loud if I had to guess. It sounds fine when I run it on my pc.
Is there anything I can do about this? Is this a known issue with gaymaker?

**Comment by xanderten50** [comment]
I've been looking for an answer to this - I take it that there's no update on this?

**Comment by xanderten50** [comment]
Just in case, the 1.7.6 update does not affect this :(

**Comment by KuromeSan** [comment]
im pretty sure this is just an issue with the GameMaker 1.4.9999 runner,

does the same thing occur on 1.4.1804? or lower?

**Comment by xanderten50** [comment]
I haven't tried it with any other version... I've only been using 1.4.99 as it was the last version, assuming this would be the best one? I can try installing 1804 and see if it makes any difference?

**Comment by xanderten50** [comment]
I installed the 1.4.1804 version of Gamemaker Studio, compiled & ran the game - then used GayMaker (set to 1.4.1804) to compile the VPK - seems to have exactly the same result; sound effects seem to be fine but music is practically muted unless I try and max out the volume?

**Comment by KuromeSan** [comment]
i dont think its something i can fix from within GayMaker, though, unless offical PSVita GM games dont have the problem or something

**Comment by xanderten50** [comment]
Fair enough, I just wish I knew how/why it happened... some sound effects are fine, while others just seem to disappear or drop so low!
